avocado/utils/disk.py

```diff
@@ -170,3 +171,67 @@ def is_root_device(device):
         if item['mountpoint'] == "/" and device == str(item['pkname']):
             return True
     return False
+
+
+def is_disk_mounted(device):
+    """
+    check if given disk is mounted or not
+    param: disk/device name
+    Returns True if the device/disk is mounted else False
+    """
+    out = process.getoutput('mount').splitlines()
+    for line in out:
```

Comment:
running os command inside python make performance degradation , using file iterator can good option , 

can be like 
```
def is_disk_mounted(device):
 with open('/proc/mounts') as mounts:
    for mount_line in mounts.readlines():
        dev, fs_dir, fs_vfstype, mount_op, _, _ = mount_line.split()
        if dev == device:
            return True
     return False
```

Reply:
sure.